diff --git a/utils/ttyd/Makefile b/utils/ttyd/Makefile
index c6733c4..92a99a5 100644
--- a/utils/ttyd/Makefile
+++ b/utils/ttyd/Makefile
@@ -43,10 +43,6 @@ endef
 define Package/ttyd/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/ttyd $(1)/usr/bin/
-	$(INSTALL_DIR) $(1)/etc/config/
-	$(INSTALL_CONF) ./files/ttyd.config $(1)/etc/config/ttyd
-	$(INSTALL_DIR) $(1)/etc/init.d/
-	$(INSTALL_BIN) ./files/ttyd.init $(1)/etc/init.d/ttyd
 endef
 
 $(eval $(call BuildPackage,ttyd))
